### ISSUE TITLE
typeIn helper does not work in IE11

### ISSUE THREAD

**Issue by john-griffin** [emberjs/ember-test-helpers#420]
Works as expected in other browsers but I get an error `TypeError: Object doesn't support this action` in IE11.

**Comment by rwjblue** [comment]
Can you share the stack trace? Any specifics on the object / action it is referring to?

**Comment by john-griffin** [comment]
Regular text input, replacing a working `fillIn` call.

```
TypeError: Object doesn't support this action
   at keyEntry (http://10.0.1.7:4200/assets/tests.js:1302:5)
   at Anonymous function (http://10.0.1.7:4200/assets/tests.js:1283:7)
   at fillOut (http://10.0.1.7:4200/assets/tests.js:1282:5)
   at Anonymous function (http://10.0.1.7:4200/assets/tests.js:1274:7)
   at tryCatcher (http://10.0.1.7:4200/assets/vendor.js:64618:7)
   at invokeCallback (http://10.0.1.7:4200/assets/vendor.js:64790:7)
   at publish (http://10.0.1.7:4200/assets/vendor.js:64776:9)
   at Anonymous function (http://10.0.1.7:4200/assets/vendor.js:57013:9)
   at invoke (http://10.0.1.7:4200/assets/vendor.js:32734:17)
   at flush (http://10.0.1.7:4200/assets/vendor.js:32654:25)
```

**Comment by rwjblue** [comment]
Some changes landed in @ember/test-helpers@1.2.0 to make fillIn go through the same path as triggerKeyEvent. I _think_ it should have addressed this issue (or `triggerKeyEvent` would have had the same problem).

**Comment by john-griffin** [comment]
Thanks, our `@ember/test-helpers` version seems to be constrained by the version of `ember-qunit` that comes with cli. Any reason the cli blueprint is still requiring `ember-qunit` `^3.4.1`?

**Comment by rwjblue** [comment]
Nope, I had opened an issue to address that a few days ago (https://github.com/ember-cli/ember-cli/issues/8395).

**Comment by john-griffin** [comment]
Ah great 😄

**Comment by btecu** [comment]
This appears to still be an issue:
![image](https://user-images.githubusercontent.com/7517745/75350878-38779380-5875-11ea-8a82-1162ef2b3c01.png)

**Comment by rwjblue** [comment]
Hmph. Anyone have time to dig in and help figure out what is wrong?

**Comment by btecu** [comment]
This will probably be fixed by #872.

**Comment by rwjblue** [comment]
Fixed a few more IE11 issues in https://github.com/emberjs/ember-test-helpers/commit/d3ccfa45597c690957af0d68be3a97d26444eb3f.

Closing this as I believe it is working now (mostly fixed by #872).
